releases/cloud.md

```diff
@@ -13,9 +13,24 @@ Get future release notes emailed to you:
 {% include marketo.html %}
 
 {{site.data.alerts.callout_version}}
-As of June 6, 2022, CockroachDB [v22.1.1](v22.1.html) is available on {{ site.data.products.dedicated }} clusters. {{ site.data.products.serverless }} clusters are running CockroachDB [v21.2.10](v21.2.html).
+As of July 6, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v21.2.12](v21.2.html) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.2](v22.1.html).
 {{site.data.alerts.end}}
 
+## July 6, 2022
+
+<h3>Console changes</h3>
+
+- The [**Connect to your cluster**](../{{site.versions["stable"]}}/connect-to-the-database.html) dialog now includes code snippets for [supported languages and tools](../{{site.versions["stable"]}}/third-party-database-tools.html).
+- The [**Connect to your cluster**](../cockroachcloud/connect-to-a-serverless-cluster.html) dialog for clusters running CockroachDB [v22.1](v22.1.html) now loads more quickly.
+- The **Databases** page now shows the recommended number of indexes for the database.
```

Comment:
We should remove "The **Databases** page now shows the recommended number of indexes for the database."

I'm assuming this refers to "Drop unused index" recommendations which is not available until 22.2.

Reply:
Adding @THardy98 to correct me here

Reply:
Correct, initially it was merged for 22.1, but after adding the table/index details pages on 22.2, we decided to version-gate everything to 22.2 to keep everything together.